test/nodejs-bundle/tcp+websockets+websocket-star.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-env mocha */
```

Comment:
Instead of creating a copy of `tcp+websockets+webrtc-star` please do try to augment the other test suite